src/tsickle.ts

```diff
@@ -888,6 +888,36 @@ class ExternsWriter extends ClosureRewriter {
       this.writeExternsFunction(name.getText(), paramNames, namespace);
     }
 
+    // Handle method declarations/signatures first, since we need to deal with overloads.
+    // TODO: Build a collection of the remaining members
+    let methods: {[id: string]: ts.MethodDeclaration[];} = {};
+    for (let member of decl.members) {
+      if (member.kind === ts.SyntaxKind.MethodSignature ||
+          member.kind === ts.SyntaxKind.MethodDeclaration) {
+        let m = <ts.MethodDeclaration>member;
+        let overloaded = methods[m.name.getText()];
```

Comment:
getText() tries to get the literal source text of the node, and it turns out to not be usually what you want.  It appears m.name is
```
    export type PropertyName = Identifier | LiteralExpression | ComputedPropertyName;
```
Maybe just handle `m.name.kind === ts.SyntaxKind.Identifier` for now and punt on the others?